src/js/video.js

```diff
@@ -147,6 +148,15 @@ videojs.hooks_ = {};
  */
 videojs.hooks = function(type, fn) {
   videojs.hooks_[type] = videojs.hooks_[type] || [];
+  if (fn && type === 'vttjsloaded') {
+    vttjsOnLoad((vttjs) => {
```

Comment:
The onload function seems a bit weird in the vttjsloader module. Maybe we should just move all the logic for that here? I don't think that it would be used anywhere else.